pyproject.toml

```diff
@@ -6,36 +6,36 @@ authors = [
     { name = "D. Seifert", email = "Domiii@users.noreply.github.com" }
 ]
 dependencies = [
-    "langchain==0.2.7",
-    "langchain-core>=0.2.7",
-    "langchain-community==0.2.7",
+    "aiosqlite>=0.20.0",
+    "colorlog>=6.8.2",
+    "faiss-cpu>=1.8.0.post1",
+    "google-auth>=2.32.0",
+    "google-cloud-speech>=2.26.1",
+    "graphviz>=0.20.3",
     "langchain-anthropic>=0.1.20",
+    "langchain-community>=0.2.7",
+    "langchain-core>=0.2.7",
+    "langchain-fireworks>=0.1.5",
     "langchain-ollama>=0.1.0",
     "langchain-openai>=0.1.15",
-    "langchain-fireworks>=0.1.5",
+    "langchain==0.2.7",
     "langgraph>=0.1.11",
-    "aiosqlite>=0.20.0",
+    "networkx>=3.3",
     "openai>=1.35.13",
     "opentelemetry-api>=1.25.0",
-    "opentelemetry-sdk>=1.25.0",
     "opentelemetry-exporter-otlp-proto-http>=1.25.0",
-    "python-dotenv>=1.0.1",
-    "unstructured>=0.14.10",
-    "faiss-cpu>=1.8.0.post1",
-    "tiktoken>=0.7.0",
-    "graphviz>=0.20.3",
-    "pylint>=3.2.5",
+    "opentelemetry-sdk>=1.25.0",
     "pathspec>=0.12.1",
     "pyaudio>=0.2.14",
-    "speechrecognition>=3.10.4",
-    "google-auth>=2.32.0",
-    "google-cloud-speech>=2.26.1",
-    "pytest>=8.3.1",
+    "pylint>=3.2.5",
     "pynput>=1.7.7",
-    "networkx>=3.3",
-    "colorlog>=6.8.2",
```

Comment:
Curious: How did you get rid of unused dependencies? Is there a `rye` command for that?

Reply:
oh I meant to call that out - I didn't remove anything.  I reordered them alphabetically.